<?xml version="1.0"?>
<launch>
    <!-- <arg name="map_file" default="$(find clean_robot)/maps/map/20250925_03-28-14.yaml"/> -->
    <arg name="open_rviz" default="true"/>
    <!-- 启动仿真环境 -->
    <include file="$(find clean_robot)/launch/pool_gazebo.launch"/>

    <!-- 启动导航系统，用作目标点路径规划，实现躲避小障碍物-->
    <include file="$(find clean_robot)/launch/my_navigation.launch">
    </include>

    <!-- 清扫路径规划器 -->
    <!-- <node pkg="clean_robot" type="coverage_planner_node" respawn="false" name="coverage_planner_node" output="screen" clear_params="true">
        <rosparam file="$(find clean_robot)/config/costmap_common_params.yaml" command="load" ns="cleaning_costmap" />
        <rosparam file="$(find clean_robot)/config/cleaning_costmap_params.yaml" command="load" />
    </node> -->
    <!-- 根据清扫的路径向导航系统发送目标点位 -->
    <!-- 设定距离当前目标点多进时候发布下一个目标点 -->
    <!-- <param name="/NextGoal/tolerance_goal" value="0.25" />
    <node pkg="clean_robot" type="publish_goal" respawn="true" name="publish_goal" output="screen" /> -->
</launch>
